src/content/docs/cheerp/11-guides/80-clientlib-migration.mdx

```diff
@@ -1,10 +1,13 @@
 ---
 title: Clientlib migration guide
+description: How to migrate from `clientlib.h` to `client.h`
```

Comment:
The description doesn't make sense. This suggests that `clientlib.h` was renamed to `client.h`, but both headers still exist and have the same purpose as they did before.